src/android/OpenTokAndroidPlugin.java

```diff
@@ -980,6 +980,22 @@ public JSONObject createDataFromStream(Stream arg1) {
             stream.put("fps", -999);
             stream.put("hasAudio", arg1.hasAudio());
             stream.put("hasVideo", arg1.hasVideo());
+
+            JSONObject videoDimensions = new JSONObject();
+            try {
+                videoDimensions.put("width", arg1.getVideoWidth());
+                videoDimensions.put("height", arg1.getVideoHeight());
+            } catch (JSONException e) {}
+            stream.put("videoDimensions", videoDimensions);
+
+            String videoType = "custom";
+            if(arg1.getStreamVideoType() == Stream.StreamVideoType.StreamVideoTypeCamera) {
+                videoType = "camera";
+            } else if(arg1.getStreamVideoType() == Stream.StreamVideoType.StreamVideoTypeScreen) {
+                videoType = "screen";
```

Comment:
Since screen sharing has not been implemented, this will not fire, but thanks for adding this in :)